lib/components/molecules/StickyHeader/tests/StickyHeader.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { StickyHeaderVanilla } from '../StickyHeader';
+
+describe('Test render <StickyHeader /> Component', () => {
```

Comment:
These tests are not enough, they are not even testing the core functionality

Reply:
Facing issue in hook testing. Will discuss it.